libcontainer/cgroups/fs/fs_test.go

```diff
@@ -295,3 +295,14 @@ func TestInvalidAbsoluteCgroupNameAndParent(t *testing.T) {
 		t.Errorf("SECURITY: cgroup path() is outside cgroup mountpoint!")
 	}
 }
+
+func TestTryDefaultCgroupRoot(t *testing.T) {
+	res := tryDefaultCgroupRoot()
+	exp := defaultCgroupRoot
+	if cgroups.IsCgroup2UnifiedMode() {
+		exp = ""
```

Comment:
why not t.skip?

Reply:
We are actually checking the case that `tryDefaultCgroupRoot()` correctly returns empty string when `/sys/fs/cgroup` is something other than cgroupv1 root. Let me add a comment about it.

Reply:
Could you add a comment? Otherwise, PR looks good!

Reply:
ping @kolyshkin

Reply:
did that already but forgot to push. rebased, pushed